src/lib/forms/Slider.svelte

```diff
@@ -74,7 +74,7 @@
     value={valueDisplayed.current}
     {min}
     {max}
-    step={step === "any" ? 0.001 : step}
+    step={step === "any" ? undefined : step}
```

Comment:
`undefined` turns into `1` which is problematic actually

Reply:
removing the attribute *is* problematic, because it will default to only allowing whole numbers